ui-manchette/src/stories/Manchette.stories.ts

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import '@osrd-project/ui-core/dist/theme.css';
+import '@osrd-project/ui-manchette/dist/theme.css';
+import { SAMPLE_DATA } from './assets/sampleData';
+
+import Manchette from '../components/Manchette';
+
+const OperationalPointListData = SAMPLE_DATA.operational_points;
+
+const meta: Meta<typeof Manchette> = {
+  component: Manchette,
+  title: 'Manchette',
```

Comment:
Group all the stories in a category "Manchette"
```suggestion
  title: 'Manchette/Manchette',
```